src/main/java/luca/task/Event.java

```diff
@@ -0,0 +1,81 @@
+package luca.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Tasks that has as specific start time and end times.
+ */
+public class Event extends Task {
+
+    /** String representation of the event start dates/times. */
+    protected LocalDateTime start;
```

Comment:
Since once a task is created, time cannot be changed, the time of event/deadline can be declared final.